src/h5web/visualizations/MappedMatrixVis.tsx

```diff
@@ -15,14 +15,8 @@ function MappedMatrixVis(props: Props): ReactElement {
   const { value, dataset, mapperState } = props;
   assertArray<number | string>(value);
 
-  return (
-    <MappedVis
-      component={MatrixVis}
-      dataset={dataset}
-      value={value}
-      mapperState={mapperState}
-    />
-  );
+  const dataArray = useMappedArray(dataset, value, mapperState);
+  return <MatrixVis dataArray={dataArray} />;
```

Comment:
You could even inline the `dataArray`. As you wish.

Reply:
Meh, I quite like it like this, especially with what's coming next 😉